lib/mediawiki.Title.phpCharToUpper.js

```diff
@@ -1,8 +1,8 @@
-"use strict";
+'use strict';
 
 // ESLint suggests unquoting some object keys, which would render the file unparseable by Opera 12.
 /* eslint-disable quote-props */
-(function() {
+(function () {
```

Comment:
idem

Reply:
I tried. Changing the IFFE to arrow causes a lint parsing error.